frontend/index.html

```diff
@@ -468,8 +529,91 @@ <h2>Contact</h2>
         const study_name = first_domain.substr(0,openpath_index);
         return study_name;
       }
+
+      var curr_lang = "en"; // Default language is set to English
+
+      function getLang(){
+
+      var selectBox = document.getElementById("buttonBox");
+      var selectedValue = selectBox.options[selectBox.selectedIndex].value;
+      curr_lang = selectedValue;
+      localStorage.setItem("langValue", curr_lang);
+
 			$(function() {
 				$(document).ready(function() {
+
+					const STUDY_NAME = _getStudyName(window.location);
+          const JOIN_CODE = `nrelopenpath://join_study?label=${STUDY_NAME}&source=github`
+
+          $.getJSON(`https://raw.githubusercontent.com/e-mission/nrel-openpath-deploy-configs/main/configs/${STUDY_NAME}.nrel-op.json`, function(data) {
+
+            // Title card
+            const PAGE_TITLE_TEXT = `${data.intro.translated_text[curr_lang].deployment_name} Join Page`
+            $('#pageTitle').text(PAGE_TITLE_TEXT);
+            document.title = PAGE_TITLE_TEXT;
+
+            // Navigation links
+            const PUBLIC_DASHBOARD_LINK = data.server.connectUrl.replace("api", "public");;
+            $('#publicDashboardLink').attr('href', PUBLIC_DASHBOARD_LINK);
+
```

Comment:
Isn't all of this copy-pasted from the default load below? What did they teach you about copy-pasting in your software engineering classes?

Reply:
Yes, everything is explained in the above comment.